client.go

```diff
@@ -0,0 +1,32 @@
+package dp_redis
+
+import (
+	"github.com/go-redis/redis"
+	"time"
+)
+
+// Redis - structure for the redis client
+type Redis struct {
```

Comment:
your editor (or, more generally, `go vet`) should warn you that this name is undesirable because users of this package would have to use something like `redis.Redis` which repeats, so consider:
```suggestion
type Client struct {
```

Reply:
Makes sense, I think I've come across a few times where I've seen something similar. Renamed in next commit.